digdag-standards/src/main/java/io/digdag/standards/command/kubernetes/KubernetesClientConfig.java

```diff
@@ -46,24 +51,49 @@ private static KubernetesClientConfig createFromSystemConfig(final Optional<Stri
         }
         final String keyPrefix = KUBERNETES_CLIENT_PARAMS_PREFIX + clusterName + ".";
         final Config extracted = validateParams(StorageManager.extractKeyPrefix(systemConfig, keyPrefix));
-        return create(clusterName,
-                extracted.get("master", String.class),
-                extracted.get("certs_ca_data", String.class),
-                extracted.get("oauth_token", String.class),
-                extracted.get("namespace", String.class));
+        if (extracted.get("use_kube_config", boolean.class, false)) {
```

Comment:
I'm wondering that 'use_kube_config' key is actually necessary or not. instead 'kube_config_path' key could be used for this condition?